deploy/overlays/prod/kustomization.yaml

```diff
@@ -40,6 +40,9 @@ configMapGenerator:
   - name: redis-config
     files:
       - redis.json
+  - name: ai-config
```

Comment:
Instead of "prod" maybe can first put these in "autopush" folder, which is an instance used for testing.